server/src/controller/user/user.controller.ts

```diff
@@ -1,25 +1,88 @@
-import { controller, httpPost, httpGet } from "inversify-express-utils";
+import { controller, httpPost, httpGet, httpPut, response, request, queryParam, httpDelete, } from "inversify-express-utils";
 import { inject } from 'inversify';
-import { Request, Response } from 'express';
-import { UserService } from "service/user";
+import * as express from 'express';
+import { UserRepository } from "service/user";
+import { User } from '../../models';
```

Comment:
as I remember we have base url option, 
please use import { User } from 'models'
please go through other places